src/qt/bitcoingui.cpp

```diff
@@ -1101,6 +1101,11 @@ void BitcoinGUI::message(const QString& title, QString message, unsigned int sty
     }
 }
 
+void BitcoinGUI::handleNonFatalCheckError(const QString& error)
+{
+    QMessageBox::warning(nullptr, "NonFatalCheckError exception", error);
```

Comment:
Would change dialog title to something more understandable like "Internal error", since if this triggers something's gone wrong like memory corruption or a bug triggered in the code

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18897#issuecomment-624710077).